### ISSUE TITLE
Performance testing and code optimization

### ISSUE THREAD

**Issue by zepumph** [phetsims/density-buoyancy-common#352]
Original performance testing was done in https://github.com/phetsims/density-buoyancy-common/issues/169. This was more of a usability-based test that QA did. But @samreid and I would like to spend a bit of time seeing if there are any large and easy optimizations that we could do based on the performance profiler.

**Comment by zepumph** [comment]
@samreid and I built a sim with `minify.mangle=false` for performance testing. We opened up the first screen and just bobbed the block in the water for 10 seconds. 5% of that js work was spent in `GSSolver.iterateEquation`. It is run 500 times to find the best approximation, as stated by `?p2FrictionIterations`. We found that that could dissappear by changing this to something like 10, but in https://github.com/phetsims/density-buoyancy-common/commit/2235ba12c8bcd326d48977fdad444dd7caed4db7 it was changed to this large number, so likely we shouldn't decrease it by an order of magnitude. We will not improve on this at all.
